buildserver/integration_test.go

```diff
@@ -0,0 +1,284 @@
+
+				"config/vendor/github.com/coreos/ignition/config/vendor/github.com/coreos/go-semver/semver/semver_test.go:287:27": "git://github.com/go-yaml/yaml?v2.2.2#yaml.go:138:6", // diff repo
+			},
+			wantXDefinition: map[string]string{
+				"config/convert.go:262:26": "git://github.com/coreos/fuze?7df4f06041d9daba45e4c68221b9b04203dff1d8#config/vendor/github.com/alecthomas/units/bytes.go:30:6 id:github.com/coreos/fuze/config/vendor/github.com/alecthomas/units/-/ParseBase2Bytes name:ParseBase2Bytes package:github.com/coreos/fuze/config/vendor/github.com/alecthomas/units packageName:units recv: vendor:true",
+			},
+		},
+		"git://github.com/golang/lint?c7bacac2b21ca01afa1dee0acf64df3ce047c28f": {
+			mode: "go",
+			pinDepReposToRev: map[string]string{
+				"https://github.com/golang/tools": "73d2e795b859a48cba2d70040c384dd1cea7e113",
+			},
+			wantHover: map[string]string{
+				"golint/golint.go:91:18": "type Linter struct", // diff pkg, same repo
+			},
+			wantDefinition: map[string]string{
+				"golint/golint.go:91:18": "git://github.com/golang/lint?c7bacac2b21ca01afa1dee0acf64df3ce047c28f#lint.go:31:6", // diff pkg, same repo
+			},
+			wantXDefinition: map[string]string{
+				"golint/golint.go:91:18": "git://github.com/golang/lint?c7bacac2b21ca01afa1dee0acf64df3ce047c28f#lint.go:31:6 id:github.com/golang/lint/-/Linter name:Linter package:github.com/golang/lint packageName:lint recv: vendor:false",
+			},
+		},
+		"git://github.com/gorilla/csrf?a8abe8abf66db8f4a9750d76ba95b4021a354757": {
+			mode: "go",
+			pinDepReposToRev: map[string]string{
+				"https://github.com/gorilla/securecookie": "c13558c2b1c44da35e0eb043053609a5ba3a1f19",
+				"https://github.com/gorilla/context":      "08b5f424b9271eedf6f9f0ce86cb9396ed337a42",
+				"https://github.com/pkg/errors":           "839d9e913e063e28dfd0e6c7b7512793e0a48be9",
+			},
+			wantHover: map[string]string{
+				"csrf.go:57:28": "type SecureCookie struct", // diff repo
+			},
+			wantDefinition: map[string]string{
+				"csrf.go:57:28": "git://github.com/gorilla/securecookie?HEAD#securecookie.go:154:6", // diff repo
+			},
+			wantXDefinition: map[string]string{
+				"csrf.go:57:28": "git://github.com/gorilla/securecookie?HEAD#securecookie.go:154:6 id:github.com/gorilla/securecookie/-/SecureCookie name:SecureCookie package:github.com/gorilla/securecookie packageName:securecookie recv: vendor:false",
+			},
+			wantXDependencies: "gorilla-csrf.json",
+		},
+		"git://github.com/golang/go?f75aafdf56dd90eab75cfeac8cf69358f73ba171": {
+			// SHA is equivalent to go1.7.1 tag, but make sure we
+			// retain the original rev spec in definition results.
+			mode: "go",
+			pinDepReposToRev: map[string]string{
+				"https://github.com/foobar/externalprintf": "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef",
+			},
```

Comment:
why was this added?

Reply:
Without this, the test code throws an error complaining about an unpinned dependency. As far as I can tell, this repo doesn't actually exist. It's imported by some test data file in go vet https://sourcegraph.com/github.com/golang/go@f75aafd/-/blob/src/cmd/vet/testdata/print.go#L17:21